auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Rule
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCUptake'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/RulesBase'
+                        - $ref: '#/definitions/RulesSCNullable'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                      required:
+                        - alias
+                        - backgroundRate
+                        - buildID
+                        - buildTarget
+                        - change_type
+                        - channel
+                        - comment
+                        - data_version
+                        - distVersion
+                        - distribution
+                        - fallbackMapping
+                        - headerArchitecture
+                        - instructionSet
+                        - locale
+                        - mapping
+                        - memory
+                        - osVersion
+                        - priority
+                        - product
+                        - rule_id
+                        - telemetry_channel
+                        - telemetry_product
+                        - telemetry_uptake
+                        - update_type
+                        - version
+                        - when
+    post:
+      summary: create scheduled change for rules.
+      description: >
+        Create scheduled change for a rule(existing or new). [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id34)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_rules_post
+      consumes:
+        - application/json
+      produces:
+        - text/html
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: "create scheduled change for rule."
+      parameters:
+        - name: change_type
+          in: query
+          description: "'change_type' value for a scheduled change."
+          type: string
+          enum:
+            - insert
+            - delete
+            - update
+          minLength: 1
+          maxLength: 20
+          x-nullable: true
+          required: false
+        - name: sc_rule_body
```

Comment:
this is a query parameter which may not be required if the change_type is passed in the request.json body